journal/src/main/scala/com/evolutiongaming/kafka/journal/Journal.scala

```diff
@@ -32,7 +32,7 @@ trait Journal[F[_]] {
   /**
    * Deletes events up to provided SeqNr
    */
-  def delete(to: DeleteTo = DeleteTo.max): F[Option[PartitionOffset]]
+  def delete(to: DeleteTo): F[Option[PartitionOffset]]
```

Comment:
Removing default value is binary incompatible change. The trait is public thus someone outside of KJ uses it, be careful.

IMO its better to keep `to: DeleteTo = DeleteTo.max` and change on next major version bump.